apps/next/src/app/app.ts

```diff
@@ -18,9 +18,9 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'x-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss'],
+  templateUrl: './app.html',
+  styleUrls: ['./app.scss'],
 })
-export class AppComponent {
+export class Next {
```

Comment:
I think it makes sense to stick to a naming convention - I am fine with dropping the Component after the name, but i'd suggest using the name of the class also in the filetype. So either rename the class to App or change the filename to next.ts ;)